src/main/java/de/tum/in/www1/artemis/repository/CourseRepository.java

```diff
@@ -49,6 +50,11 @@ public interface CourseRepository extends JpaRepository<Course, Long> {
     @Query("select distinct course from Course course where course.studentGroupName like :name")
     Course findCourseByStudentGroupName(@Param("name") String name);
 
+    @Query("""
+            SELECT CASE WHEN (count(c) > 0)  THEN true ELSE false END
+            FROM Course c WHERE c.id = :#{#courseId} AND c.courseInformationSharingConfiguration IN :#{#values}""")
```

Comment:
```suggestion
            SELECT CASE WHEN (count(c) > 0) THEN true ELSE false END
            FROM Course c
            WHERE c.id = :#{#courseId}
                AND c.courseInformationSharingConfiguration IN :#{#values}
            """)
```
(reformatting change only)